### DISCUSSION TITLE
Fail to start Zigbee2mqtt after updates

### DISCUSSION THREAD

**Discussion by ls500h** [Koenkk/zigbee2mqtt#17355] [Devices]
I use a Rpi3 running Sonoff Zigbee 3, Node Red and Mosquitto - I shut it down to backup the SD card - and found the card was corrupt :(  So I grabbed the old back up SDCard, (made from pre the Sonoff Zigbee 3 installation 6 months ago)  and tried to use it. As newer version of Node/ nodeJS or was it NPM was needed so I have updated those to
node v19,9.0
nodejs v10.24.1
npm 9.6.3

I suspect that I missed a comment in the update process that told me that a "location" had changed but cannot find that comment again.

Now when I npm start from within /opt/zigbee2mqtt i get

info  2023-04-16 10:11:14: Logging to console and directory: '/opt/zigbee2mqtt/data/log/2023-04-16.10-11-14' filename: log.txt
info  2023-04-16 10:11:14: Starting Zigbee2MQTT version 1.30.3 (commit #24c6b2e)
info  2023-04-16 10:11:14: Starting zigbee-herdsman (0.14.103)
error 2023-04-16 10:12:22: Error while starting zigbee-herdsman
error 2023-04-16 10:12:22: Failed to start zigbee
error 2023-04-16 10:12:22: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
error 2023-04-16 10:12:22: Exiting...
error 2023-04-16 10:12:23: Error: network commissioning timed out - most likely network with the same panId or extendedPanId already exists nearby
    at ZnpAdapterManager.beginCommissioning (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/manager.ts:347:23)
    at ZnpAdapterManager.start (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/manager.ts:92:17)
    at Controller.start (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/controller/controller.ts:132:29)
    at Zigbee.start (/opt/zigbee2mqtt/lib/zigbee.ts:58:27)
    at Controller.start (/opt/zigbee2mqtt/lib/controller.ts:101:27)
    at start (/opt/zigbee2mqtt/index.js:107:5)

Sonoff is usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_100f74db8612ec11804f23c7bd930c07-if00-port0
Config: /opt/zigbee2mqtt/data/configuration.yaml is
homeassistant: false
permit_join: true
mqtt:
  base_topic: zigbee2mqtt
  server: mqtt://localhost
serial:
  port: /dev/ttyUSB0

frontend: true

What am I doing wrong?

**Comment by ls500h** [comment]
And if I run sudo journalctl -u zigbee2mqtt.service -f
i get a consatnt repeat of:
Apr 16 11:30:08 raspberrypi systemd[1]: zigbee2mqtt.service: Service RestartSec=10s expired, scheduling restart.
Apr 16 11:30:08 raspberrypi systemd[1]: zigbee2mqtt.service: Scheduled restart job, restart counter is at 3650.
Apr 16 11:30:08 raspberrypi systemd[1]: Stopped zigbee2mqtt.
Apr 16 11:30:08 raspberrypi systemd[1]: Started zigbee2mqtt.
Apr 16 11:30:09 raspberrypi zigbee2mqtt[28438]: /opt/zigbee2mqtt/node_modules/rimraf/node_modules/minimatch/dist/cjs/index.js:208
Apr 16 11:30:09 raspberrypi zigbee2mqtt[28438]:     options;
Apr 16 11:30:09 raspberrypi zigbee2mqtt[28438]:            ^
Apr 16 11:30:09 raspberrypi zigbee2mqtt[28438]: SyntaxError: Unexpected token ;
Apr 16 11:30:09 raspberrypi zigbee2mqtt[28438]:     at Module._compile (internal/modules/cjs/loader.js:723:23)
Apr 16 11:30:09 raspberrypi zigbee2mqtt[28438]:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
Apr 16 11:30:09 raspberrypi zigbee2mqtt[28438]:     at Module.load (internal/modules/cjs/loader.js:653:32)
Apr 16 11:30:09 raspberrypi zigbee2mqtt[28438]:     at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
Apr 16 11:30:09 raspberrypi zigbee2mqtt[28438]:     at Function.Module._load (internal/modules/cjs/loader.js:585:3)
Apr 16 11:30:09 raspberrypi zigbee2mqtt[28438]:     at Module.require (internal/modules/cjs/loader.js:692:17)
Apr 16 11:30:09 raspberrypi zigbee2mqtt[28438]:     at require (internal/modules/cjs/helpers.js:25:18)
Apr 16 11:30:09 raspberrypi zigbee2mqtt[28438]:     at Object.<anonymous> (/opt/zigbee2mqtt/node_modules/rimraf/node_modules/minimatch/dist/cjs/index-cjs.js:5:36)
Apr 16 11:30:09 raspberrypi zigbee2mqtt[28438]:     at Module._compile (internal/modules/cjs/loader.js:778:30)
Apr 16 11:30:09 raspberrypi zigbee2mqtt[28438]:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)

**Reply by Koenkk** [reply]
You are probably using a non-supported node.js version, updating to node.js 18 should fix your issue.

**Comment by yuval1983** [comment]
I have the exact same - I did an update and then I got the same message - before the update to the latest version everything worked wonderfully

	  
	 Zigbee2MQTT:info   Starting Zigbee2MQTT version 1.30.3 (commit #24c6b2e)

Zigbee2MQTT:info  Starting zigbee-herdsman (0.14.103)

Zigbee2MQTT:error  Error while starting zigbee-herdsman

Zigbee2MQTT:error  Failed to start zigbee

Zigbee2MQTT:error : Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions

Zigbee2MQTT:error Exiting...

Zigbee2MQTT:error  Error: Error while opening serialport 'Error: Error: No such file or directory, cannot open /dev/ttyUSB1'

    at SerialPort.<anonymous> (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/znp/znp.ts:146:28)

    at SerialPort._error (/app/node_modules/@serialport/stream/dist/index.js:76:22)

    at /app/node_modules/@serialport/stream/dist/index.js:112:18

**Reply by Koenkk** [reply]
The path to the adapter is wrong, see https://www.zigbee2mqtt.io/guide/installation/01_linux.html#determine-location-of-the-adapter-and-checking-user-permissions how to figure it out

**Comment by yuval1983** [comment]
/docker/zigbee2mqtt/data$ sudo nano configuration.yaml

I completely forgot there was this file! lol

Now I'm back to living
